public/pages/CreateDetector/components/DefineDetector/components/DetectorSchedule/Interval.tsx

```diff
@@ -29,7 +29,7 @@ export interface IntervalState {
 
 export class Interval extends React.Component<IntervalProps, IntervalState> {
   state = {
-    isIntervalValid: true,
+    isIntervalValid: Number.isInteger(this.props.schedule.period.interval),
```

Comment:
why are we validating this? it should be integer only right as per this -> https://github.com/amsiglan/security-analytics-dashboards-plugin/blob/964af5ba98b6b0d788b3c829a58984deea6341f5/models/interfaces.ts#L34 or it can be non-integer value als?

Reply:
If the user deletes the value it becomes `NaN`